docs/analyze.md

```diff
@@ -1,124 +1,141 @@
 # Monitor and analyze test run
-This document will walk you through enabling data collection for a test run. We will
-start with brief overview of data collectors, followed by instructions for [code
-coverage][coverage].
+This document will walk you through enabling data collection for a test run covering:
+1. A brief overview of DataCollectors.
+2. Configuring DataCollectors in TPv2.
+3. Key differences for using DataCollectors in TPv2 v/s TPv1.
+3. Instructions for using [code coverage][coverage].
 
 > **Version note:**
 >
-> Data collectors are supported on test platform `15.3.0` onwards. It is part of
+> DataCollectors are supported on test platform `15.3.0` onwards. It is part of
 > VS 2017 15.3 and dotnet-cli 2.0.0 builds.
```

Comment:
Pl add an appendix section clarifying >= build numbers where it is available and how to get that info from help->about in VS and dotnet --help etc.